src/components/color-picker/ColorPicker.tsx

```diff
@@ -21,10 +21,16 @@ export const ColorPicker: React.FC<IColorPicker> = ({ initColor, onChange }) =>
   const hexColor = useMemo(() => (a === 1 ? `#${rgbHex(r, g, b)}` : `#${rgbHex(r, g, b, a)}`), [r, g, b, a]);
```

Comment:
@nicolasshiken  try to remove the memo here and you will not have the issue - just compute the hex on demand where you need it  - then you will not need the useEffect below as well